go.mod

```diff
@@ -1,3 +1,21 @@
+//
```

Comment:
once we add a new dependency, this file will be auto changed by go modules. 
in my local testing, i did go mod tidy as well. looks ok so far.

Reply:
Mod files support comments in the file based on the specs. It should not remove any comment lines when you run a `go mod`
See: https://golang.org/cmd/go/#hdr-The_go_mod_file